content_handler/software_rasterizer.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_CONTENT_HANDLER_SOFTWARE_RASTERIZER_H_
+#define FLUTTER_CONTENT_HANDLER_SOFTWARE_RASTERIZER_H_
+
+#include <memory>
+#include "apps/mozart/services/buffers/cpp/buffer_producer.h"
+#include "flutter/content_handler/rasterizer.h"
+#include "flutter/flow/compositor_context.h"
+#include "lib/ftl/macros.h"
+
+namespace flutter_runner {
+
+class SoftwareRasterizer : public Rasterizer {
+ public:
+  SoftwareRasterizer();
+
+  ~SoftwareRasterizer();
```

Comment:
Elsewhere you have `std::unique_ptr<Rasterizer>`.  That makes it seem like `Rasterizer` needs a virtual destructor.  Otherwise, when that `unique_ptr` goes out of scope, `~Rasterizer` will be called but `~SoftwareRasterizer` will not be called.